third_party/arcs/proto/manifest.proto

```diff
@@ -109,9 +113,11 @@ message TypeProto {
     SingletonTypeProto singleton = 7;
     CountTypeProto count = 8;
     ListTypeProto list = 9;
+    NullableTypeProto nullable = 13;
   }
-  bool optional = 10;
   RefinementExpressionProto refinement = 11;
+  repeated AnnotationProto annotations = 12;
```

Comment:
does the parser still work correctly?

Reply:
It appears to on the tests that we are running. I think that all comes down to the fact that most of the changes in `manifest.proto` have been append-only changes, only adding proto fields. That means that the parser, using an old version of the proto, just fills in a subset of the possible fields.

The one exception is the `optional` field. Because we aren't relying upon the `optional` field in our code nor in any of the examples that we are currently compiling, we seem to be getting away with that one as well.